zulipterminal/ui_tools/boxes.py

```diff
@@ -368,25 +369,35 @@ def reactions_view(self, reactions: List[Dict[str, Any]]) -> Any:
         if not reactions:
             return ''
         try:
-            reacts = defaultdict(int)  # type: Dict[str, int]
-            custom_reacts = defaultdict(int)  # type: Dict[str, int]
+            std_stats = Counter()  # type: typing.Counter[str]
+            custom_stats = Counter()  # type: typing.Counter[str]
             for reaction in reactions:
                 if reaction['reaction_type'] == 'unicode_emoji':
-                    reacts[reaction['emoji_code']] += 1
+                    std_stats[reaction['emoji_code']] += 1
                 else:
                     # Includes realm_emoji and zulip_extra_emoji
-                    custom_reacts[":"+reaction['emoji_name']+":"] += 1
-            dis = [
-                '\\U' + '0'*(8-len(emoji)) + emoji + ' ' + str(reacts[emoji]) +
-                ' ' for emoji in reacts]
-            emojis = ''.join(e.encode().decode('unicode-escape') for e in dis)
-            custom_emojis = ''.join(
-                ['{} {} '.format(r, custom_reacts[r]) for r in custom_reacts])
+                    custom_stats[":"+reaction['emoji_name']+":"] += 1
```

Comment:
[suggestion] We could `.format` here to make this clearer; `':{}:'.format(reaction['emoji_name'])`.

Reply:
This is a corner case where small string addition could be clearer/faster. When we are only working with python3.6 this may even be less clearer with f-strings.

However, I've pushed a revised version which avoids this completely, since we don't need this in the stats part, only really in the series of strings.

Reply:
Thanks for the explanation. :+1: